renku/core/utils/urls.py

```diff
@@ -39,3 +40,11 @@ def url_to_string(url):
         return url
 
     raise ValueError('url value not recognized')
+
+
+def remove_credentials(url):
+    """Remove username and password from a URL."""
+    if url is None:
+        return ''
```

Comment:
why change type?

Reply:
This is actually fixing the type. When url is `None` then `.geturl()` returns `bytes` (`b''`) instead of normal return value of `str`.